orchagent/port/porthlpr.h

```diff
@@ -6,6 +6,22 @@
 #include <string>
 
 #include "portcnt.h"
+#include "portschema.h"
+
+static const std::unordered_map<std::string, sai_port_fec_mode_t> portFecMap =
+{
+    { PORT_FEC_NONE, SAI_PORT_FEC_MODE_NONE },
+    { PORT_FEC_RS,   SAI_PORT_FEC_MODE_RS   },
+    { PORT_FEC_FC,   SAI_PORT_FEC_MODE_FC   },
+    { PORT_FEC_AUTO, SAI_PORT_FEC_MODE_NONE }
```

Comment:
@dgsudharsan imho, this trick is used just to overcome properties parsing issue. In fact, this has nothing to do with a set of FEC configuration attributes. I understand that FEC override is tightly coupled with a FEC configuration itself, and i am ok to combine this one with the rest in terms of Config DB schema, but parsing/analysis logic should use a separate containers to store the data